techniques/system/common/1.0/restart-services.st

```diff
@@ -81,7 +81,7 @@ bundle agent restart_services
     # We cannot detect which one is used for Rudder
     !windows.!aix.!solaris::
       "run_syslog"      usebundle => disable_reporting;
-      "run_syslog"      usebundle => _method_reporting_context("Make sure syslog service runs", "None");
+      "run_syslog"      usebundle => _method_reporting_context("Log system running", "None");
```

Comment:
All other reports about syslog service are on "Log system for reports", why changing it?

Reply:
because i made a mistake :) 
it is a new component though, so we need a name stating that it's running
how about "Log system for reports running"